### PR TITLE
add empty upstream info and see also section

### LINKED ISSUE #4752: Composer Fundamentals and Workflows Doc Update
Re: [Composer Fundamentals and Workflows](https://pantheon.io/docs/composer/)

Priority: Medium

## Issue Description:

I've had a few Pantheon customers ask, "How do I change my upstream?" or "How do I stop those Drupal 8 updates from showing up? I'm doing it via Composer already." I haven't found many other pages that talk about changing the upstream to empty, so I point them here.

Today, I told a customer that the "owner" of the site needs to run the "terminus site:upstream:set" command and they pointed out that it doesn't mention that here. I believe that is the case that a team member cannot run this command, only the site owner.

## Suggested Resolution

Maybe just add a note under this command to point out that this has to be done by the Site Owner?

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #4752

## Effect
PR includes the following changes:
- Adds a link to instructions to set an empty upstream, and lists roles with that permission.
- Adds a **See Also** section linking to the D8 Composer guide.

## Remaining Work
- [x] :+1: from @dficker
- [ ] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Archive from **Done** in [Main Project](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/composer.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ A Custom Upstream based off Pantheon's example repositories would need to commit
 
 This workflow has one very serious shortcoming, that is site-specific dependencies are likely to cause a lot of conflicts. The practical use case for this workflow is for a large group of sites that require a single set of dependencies. You should only use this method if you don’t intend to use site specific themes, modules, or plugins downstream.
 
+You can also prevent upstream updates by [setting an empty upstream](/docs/guides/composer-convert/#change-upstreams). This action is available to Site Owner, Organization Administrator, and Users in Charge roles.
```
Comment (dficker) [inline source/content/composer.md]: Hmm... I didn't know Org Admins could update them as well. Interesting. Just if there's an Owning Org, or also a Supporting Org Admin? And this has been verified? As far as I knew, only Site Owner or User In Charge could do so.

Comment (alexfornuto) [reply source/content/composer.md]: @EdwardAngert please verify or edit.

Comment (EdwardAngert) [reply source/content/composer.md]: lines up with [Create a Custom Upstream
](https://pantheon.io/docs/create-custom-upstream/) and Product expectations 👍

Comment (dficker) [reply source/content/composer.md]: Cool with me. I think I just misread it and thought it meant Supporting Org Admins, which I was surprised about. But now that I look at it again and the other Doc, I get the meaning.

**Pull request merged**